spec.html

```diff
@@ -37417,67 +37419,121 @@ <h1>Runtime Semantics: QuoteJSONString ( _value_ )</h1>
         <emu-alg>
           1. Let _product_ be the String value consisting solely of the code unit 0x0022 (QUOTATION MARK).
           1. For each code unit _C_ in _value_, do
-            1. If _C_ is the code unit 0x0022 (QUOTATION MARK) or the code unit 0x005C (REVERSE SOLIDUS), then
-              1. Set _product_ to the string-concatenation of _product_ and the code unit 0x005C (REVERSE SOLIDUS).
-              1. Set _product_ to the string-concatenation of _product_ and _C_.
-            1. Else if _C_ is the code unit 0x0008 (BACKSPACE), the code unit 0x000C (FORM FEED), the code unit 0x000A (LINE FEED), the code unit 0x000D (CARRIAGE RETURN), or the code unit 0x0009 (CHARACTER TABULATION), then
-              1. Set _product_ to the string-concatenation of _product_ and the code unit 0x005C (REVERSE SOLIDUS).
-              1. Let _abbrev_ be the String value corresponding to the value of _C_ as follows:
-                <table class="lightweight">
-                  <tbody>
-                  <tr>
-                    <td>
-                      BACKSPACE
-                    </td>
-                    <td>
-                      `"b"`
-                    </td>
-                  </tr>
-                  <tr>
-                    <td>
-                      FORM FEED (FF)
-                    </td>
-                    <td>
-                      `"f"`
-                    </td>
-                  </tr>
-                  <tr>
-                    <td>
-                      LINE FEED (LF)
-                    </td>
-                    <td>
-                      `"n"`
-                    </td>
-                  </tr>
-                  <tr>
-                    <td>
-                      CARRIAGE RETURN (CR)
-                    </td>
-                    <td>
-                      `"r"`
-                    </td>
-                  </tr>
-                  <tr>
-                    <td>
-                      CHARACTER TABULATION
-                    </td>
-                    <td>
-                      `"t"`
-                    </td>
-                  </tr>
-                  </tbody>
-                </table>
-              1. Set _product_ to the string-concatenation of _product_ and _abbrev_.
+            1. If _C_ matches any entry in <emu-xref href="#table-json-single-character-escapes"></emu-xref>, then
```

Comment:
I would say `If _C_ is listed in the Code Unit Value column of <emu-xref href="#table-json-single-character-escapes"></emu-xref>`.